console/frontend/src/main/frontend/src/app/views/status/status.component.html

```diff
@@ -84,15 +85,21 @@
     <div class="col-lg-12 tabs-container">
       <ul class="nav nav-tabs" role="menu" uib-dropdown-menu="">
         <li [ngClass]="{ active: selectedConfiguration === 'All' }">
-          <a (click)="changeConfiguration('All')">All</a>
+          <a
+            [attr.data-cy-status-select-configuration]="'All'"
+            (click)="changeConfiguration('All')"
+            >All</a
+          >
         </li>
         <li
           *ngFor="let configuration of configurations"
           [ngClass]="{ active: selectedConfiguration === configuration.name }"
         >
-          <a (click)="changeConfiguration(configuration.name)">{{
-            configuration.name
-          }}</a>
+          <a
+            [attr.data-cy-status-select-configuration]="configuration.name"
```

Comment:
See https://stackoverflow.com/questions/59453410/angular-adding-cypress-data-cy-attribute - explains that dynamic binding should be done like this.

Reply:
This is correct since its binding a property in an object